app/src/converter.js

```diff
@@ -5,14 +5,29 @@ const path = require('path');
 
 const logger = require('./logger');
 const rpc = require('./weh-rpc');
+const findexe = require('./findexecutable');
 
 const exec_dir = path.dirname(process.execPath);
-let ffmpeg = path.join(exec_dir, "ffmpeg");
-let ffprobe = path.join(exec_dir, "ffprobe");
+let ffmpeg = ensureProgramExt(path.join(exec_dir, "ffmpeg"));
+let ffprobe = ensureProgramExt(path.join(exec_dir, "ffprobe"));
 
-if (os.platform() == "win32") {
-  ffmpeg += ".exe";
-  ffprobe += ".exe";
+function ensureProgramExt(programPath) {
+  if (os.platform() == "win32") {
+    return programPath + ".exe";
+  }
+  return programPath;
+}
+
+if (!findexe.fileExistsSync(ffmpeg)) {
+  ffmpeg = findexe.findExecutableFullPath("ffmpeg")
```

Comment:
On Windows, I don't sure if I should search `ffmpeg` or `ffmpeg.exe` in the PATH...

Or maybe Windows user should not use `--no-ship-ffmpeg` flag in `build.sh` at all...?

Reply:
See later discussion.